app/scripts/modules/kubernetes/src/v2/pipelines/stages/runJob/KubernetesV2RunJobStageConfig.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+
+import { IStageConfigProps, AccountService, YamlEditor, yamlDocumentsToString, IAccount } from '@spinnaker/core';
+
+import { ManifestBasicSettings } from 'kubernetes/v2/manifest/wizard/BasicSettings';
+
+export interface IKubernetesRunJobStageConfigState {
+  credentials: IAccount[];
+  rawManifest?: string;
+}
+
+export class KubernetesV2RunJobStageConfig extends React.Component<IStageConfigProps> {
+  public state: IKubernetesRunJobStageConfigState = {
+    credentials: [],
+  };
+
+  public constructor(props: IStageConfigProps) {
```

Comment:
Since we aren't using `props` or `this` in the constructor, can remove it and simplify to just initialize state like this:
`public state = { credentials: [] }`